diff --git a/dist/plugins/takumi.js b/dist/plugins/takumi.js
index a33ed9a..f0514e2 100644
--- a/dist/plugins/takumi.js
+++ b/dist/plugins/takumi.js
@@ -14,8 +14,8 @@ function takumiPlugin(options = {}) {
 	let basePath;
 	function slugsToImagePath(slugs, lang) {
 		const segments = [...slugs];
-		if (segments.length === 0) segments.push("index.webp");
-		else segments[segments.length - 1] += ".webp";
+		if (segments.length === 0) segments.push("index.png");
+		else segments[segments.length - 1] += ".png";
 		return {
 			staticPath: lang ? [lang, ...segments] : segments,
 			pathname: joinPathname(lang ?? "", basePath, ...segments)
@@ -24,7 +24,7 @@ function takumiPlugin(options = {}) {
 	function imagePathToSlugs(segs) {
 		if (segs.length === 0) return segs;
 		const slugs = [...segs];
-		slugs[slugs.length - 1] = slugs[slugs.length - 1].replace(/\.webp$/, "");
+		slugs[slugs.length - 1] = slugs[slugs.length - 1].replace(/\.png$/, "");
 		if (slugs.length === 1 && slugs[0] === "index") slugs.pop();
 		return slugs;
 	}
@@ -35,10 +35,15 @@ function takumiPlugin(options = {}) {
 			basePath = options.basePath ?? (renderMode === "dynamic" ? "/_takumi" : "/");
 			(this.data["core:page-meta"] ??= []).push((page) => {
 				const pathname = slugsToImagePath(page.slugs, page.locale).pathname;
+				const imageUrl = this.siteConfig.baseUrl ? new URL(pathname, this.siteConfig.baseUrl).href : pathname;
 				return /* @__PURE__ */ jsxs(Fragment, { children: [
 					/* @__PURE__ */ jsx("meta", {
 						property: "og:image",
-						content: this.siteConfig.baseUrl ? new URL(pathname, this.siteConfig.baseUrl).href : pathname
+						content: imageUrl
+					}),
+					/* @__PURE__ */ jsx("meta", {
+						name: "twitter:image",
+						content: imageUrl
 					}),
 					/* @__PURE__ */ jsx("meta", {
 						property: "og:image:width",
@@ -68,7 +73,7 @@ function takumiPlugin(options = {}) {
 						width,
 						height,
 						...options,
-						format: "webp"
+						format: "png"
 					});
 				}
 			});
