diff --git a/dist/providers/npm.js b/dist/providers/npm.js
index a9be7d3aa5afa44a51cef9ab417fabce1abc4908..2ee1b971b92f7ca49a826125d02539cc0c3cc344 100644
--- a/dist/providers/npm.js
+++ b/dist/providers/npm.js
@@ -81,7 +81,7 @@ var NpmRegistryClient = class {
 	async publish(pkg, { packageStore }) {
 		const distTag = packageStore.npm?.distTag;
 		if (this.client === "bun") {
-			const tarball = "pkg.tgz";
+			const tarball = path.resolve(pkg.path, "pkg.tgz");
 			const packResult = await x("bun", [
 				"pm",
 				"pack",
